stylize.py

```diff
@@ -34,43 +34,71 @@ def stylize(network, initial, content, styles, iterations,
     content_features = {}
     style_features = [{} for _ in styles]
 
+    vgg_weights, vgg_mean_pixel = vgg.load_net(network)
+    
+    layer_weight = 1.0
+    STYLE_LAYERS_WEIGHTS = {}
```

Comment:
Fixed in https://github.com/anishathalye/neural-style/pull/50/commits/bd4fe2a18035a6b7fd273879a9bbeddda3104f8d